pkg/cosign/bundle/bundle.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bundle
+
+import (
+	"github.com/sigstore/rekor/pkg/generated/models"
+)
+
+type Bundle struct {
+	VerificationData
+	VerificationMaterial
+}
+
+// VerificationData contains extra data that can be used to verify things
+// such as transparency logs and timestamped verifications.
+type VerificationData struct {
+	// RekorPayload holds metadata about recording a Signature's ephemeral key to
+	// a Rekor transparency log.
+	// Use Payload instead of TransparencyLogEntry to keep backwards compatibility.
+	Payload RekorPayload
+
+	// TimestampVerificationData holds metadata about a timestamped verification.
+	TimestampVerificationData
+}
+
+// VerificationMaterial captures details on the materials used to verify
+// signatures or any additional timestamped verifications.
+type VerificationMaterial struct {
+	// A chain of X.509 certificates.
+	CertBytes []byte
```

Comment:
I'd expect this to be a `crypto/x509.Certificate`

I guess really...is this struct a *data format* or an object? My preference is always to move parsing as early as possible so we don't have to handle parse errors in the middle of the program logic.